### ISSUE TITLE
RenderToTexture Asset does not work correctly when performed more than once in the same frame.

### ISSUE THREAD

**Issue by ChameleonO2** [RileyGuy/ReFract#3]
If more than two pulses are skipped in the same frame for Logix in the RenderToTexture Asset, the image will not be rendered properly.

Photos taken in the same frame will not be output correctly, and the next time you take a normal photo, the previous render texture will be affected by the camera that tried to render the last time.

Attached is the item URL for reproduction.

It may not come across well because of the use of DeepL.

ReFract Version 1.1.1 

neosdb:///daf74600f19aa71ad137718045a6016f1ad4128c0dccd5942103c4c1ebadae0c.7zbson

**Comment by BlueCyro** [comment]
Yep, I know exactly why this happens. This is due to an awful, terrible, disgusting hack I did to ferry data into a function somewhere. ReFract is old enough at this point that I'm going to have to re-write it. When that'll be, I'm unsure.
